tests/test_dependents.py

```diff
@@ -6,8 +6,8 @@
 from zfit.core.testing import setup_function, teardown_function, tester
 
 
-def test_get_dependents_ordered():
-    params = [zfit.Parameter(f'param{i}', i) for i in range(4)]
+def test_get_dependents_is_deterministic():
+    parameters = [zfit.Parameter(f'param{i}', i) for i in range(4)]
```

Comment:
Detail: inside we use usually params and param only for the naming, a common abbreviation.

Reply:
but make sense here as well, so perfectly fine actually, sorry!